### DISCUSSION TITLE
Can't communicate with bot over direct messages

### DISCUSSION THREAD

**Discussion by SamusAranX** [bwmarrin/discordgo#1309] [Q&A]
<img width="565" alt="screenshot" src="https://user-images.githubusercontent.com/676069/210283675-b838b7e6-1b78-4ad7-b619-cbb4d2691720.png">

I have a bot with a number of slash commands that all work flawlessly, but I can't work out how to make it accept and react to direct messages. From what I've seen, the usual handler for DMs would be the MessageCreate one, but as you can see in the screenshot above, I can't even get to that point. The bot intents are set up like this, so in theory it should be capable of dealing with DMs:

```go
b.session.Identify.Intents = discordgo.IntentsAll ^ discordgo.IntentGuildPresences
```

Attempting to send the bot a friend request results in this error message in the Discord client: 
<img width="392" alt="image" src="https://user-images.githubusercontent.com/676069/210283843-0807c4f9-0a0f-4cf4-b1b8-610201766825.png">

Has anyone built a bot with working DM support? How did you do it?

**Comment by DitchTurtL** [comment]
There is a toggle in the Developer Portal, on the Bot page for Message Content Intent. Is that On?
